### DISCUSSION TITLE
Sidekiq 如何处理异步任务 · /sidekiq

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#251] [Announcements]
https://draveness.me/sidekiq

**Comment by ebmyth** [comment]
我现在有个不解之处。我们有好几台服务器，我们启动sidekiq之后，他通过什么知道是那几台服务器在一起协作处理任务呢

**Comment by draveness** [comment]
> @ebmyth
> 我现在有个不解之处。我们有好几台服务器，我们启动sidekiq之后，他通过什么知道是那几台服务器在一起协作处理任务呢

他不知道，这些服务器是自己从队列里面拉任务处理的

**Comment by Gaolz** [comment]
感谢作者大大一步一步分析得这么详细，很有收获。
